openvdb_ax/openvdb_ax/compiler/VolumeExecutable.cc

```diff
@@ -1387,7 +1387,7 @@ VolumeExecutable::getActiveTileStreaming(const std::string& name,
                      const ast::tokens::CoreType& type) const
 {
     assert(mAttributeRegistry);
-    if (mSettings->mActiveTileStreaming == VolumeExecutable::Streaming::AUTO) {
+    if (mSettings->mActiveTileStreaming.get() == VolumeExecutable::Streaming::AUTO) {
```

Comment:
assume these were causing build issues too? (although not yet released hence no mention in release notes)

Reply:
Yes for some reason MSVC doesn't like this

https://github.com/AcademySoftwareFoundation/openvdb/runs/7822777048?check_suite_focus=true